gradle.properties

```diff
@@ -1,2 +1,6 @@
 kotlin.code.style=official
 kotlin.js.generate.executable.default=false
+
+org.gradle.caching=true
+org.gradle.jvmargs=-Xmx4g -Dfile.encoding=UTF-8
+org.gradle.parallel=true
```

Comment:
Please give explanations of these parameters separately. The target machine to run this build on is unknown, so we should not specify memory parameters.